packages/fx-core/src/plugins/solution/fx-solution/v2/deploy.ts

```diff
@@ -197,15 +199,38 @@ export async function deploy(
   ctx.logProvider.info(getLocalizedString("core.deploy.startNotice", PluginDisplayName.Solution));
   const result = await executeConcurrently(thunks, ctx.logProvider);
 
+  const botTroubleShootLink =
+    "https://aka.ms/teamsfx-bot-help#how-can-i-troubleshoot-issues-when-teams-bot-isnt-responding-on-azure";
+  const botTroubleShootDesc = getLocalizedString("core.deploy.botTroubleShoot");
+  const botTroubleShootLearnMore = getLocalizedString("core.deploy.botTroubleShoot.learnMore");
   if (result.kind === "success") {
     if (inAzureProject) {
-      const msg = getLocalizedString("core.deploy.successNotice", ctx.projectSetting.appName);
-      ctx.logProvider.info(msg);
-      ctx.userInteraction.showMessage("info", msg, false);
+      let msg = getLocalizedString("core.deploy.successNotice", ctx.projectSetting.appName);
+      // Append a new sentence for bot trouble shootting.
+      if (inBotProject) {
+        msg += ` ${botTroubleShootDesc}`;
+      }
+      ctx.logProvider.info(msg + ` ${botTroubleShootLearnMore}: ${botTroubleShootLink}.`);
```

Comment:
What about just msg + link? why don't add Lean more message again even when it already has troubleshoot desc?

Reply:
As you can see from the screenshots listed in desc, ${botTroubleShootLearnMore} is just the text of `Learn more` which is not duplicated. `Learn more` is placed ahead to guide the real link which is kind of mapping the content of `Learn more` button in the UI flow.